onnx/defs/generator/defs.cc

```diff
@@ -896,4 +896,68 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
+static const char* Bernoulli_ver15_doc = R"DOC(
+    Draws binary random numbers (0 or 1) from a Bernoulli distribution. The input tensor should be
+    a tensor containing probabilities to be used for drawing the binary random number.
+    )DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    Bernoulli,
+    15,
+    OpSchema()
+        .SetDoc(Bernoulli_ver15_doc)
+        .Attr(
+            "seed",
+            "(Optional) Seed to the random generator, if not specified we will auto generate one.",
+            AttributeProto::FLOAT,
+            OPTIONAL_VALUE)
+        .Attr(
+            "dtype",
+            "The data type for the elements of the output tensor. if not specified, we will use "
+            "the data type of the input tensor.",
+            AttributeProto::FLOAT,
```

Comment:
dtype should probably be an Int, not float.

Reply:
Thanks for this comment. I tried to run a simple example of bernouli in torch as below:
 ```python
>>> a = torch.Tensor([0, 1])
>>> torch.bernoulli(a).type()
'torch.FloatTensor'
```
So the dtype should be float too?

Reply:
The attribute is used to indicate what the type of the output should be (e.g., float, or double, or float16, etc.) But the attribute-value is an integer value from the enumeration TensorProto::DataType  (see https://github.com/onnx/onnx/blob/08afa047e90c33357bb6a13b7313105859bdd2c4/onnx/onnx.in.proto#L461 ). See, for example, this: https://github.com/onnx/onnx/blob/08afa047e90c33357bb6a13b7313105859bdd2c4/onnx/defs/generator/defs.cc#L411